conformance/README.md

```diff
@@ -13,66 +14,80 @@ List available commands:
 ```bash
 $ make
 
+build-and-load-images          Build NKG container and load it and NGINX container on configured kind cluster
 build-test-runner-image        Build conformance test runner image
+cleanup-conformance-tests      Clean up conformance tests fixtures
 create-kind-cluster            Create a kind cluster
 delete-kind-cluster            Delete kind cluster
 help                           Display this help
-install-nkg                    Install NKG with provisioner on configured kind cluster
-prepare-nkg                    Build and load NKG container on configured kind cluster
+install-nkg-edge               Install NKG with provisioner from edge on configured kind cluster
+install-nkg-local-build        Install NKG from local build with provisioner on configured kind cluster
+preload-nginx-container        Preload NGINX container on configured kind cluster
+prepare-nkg-dependencies       Install NKG dependencies on configured kind cluster
 run-conformance-tests          Run conformance tests
+undo-image-update              Undo the NKG image name and tag in deployment manifest
 uninstall-nkg                  Uninstall NKG on configured kind cluster
-update-test-kind-config        Update kind config
 ```
+
+**Note:** The following variables are configurable when running the below `make` commands:
+
+| Variable  | Default | Description |
```

Comment:
nit:
is there a way to control the width of the column?  the render is a bit hard to read (the table needs to be scrolled horizontally to read the descriptions of the variables):
<img width="1192" alt="image" src="https://github.com/nginxinc/nginx-kubernetes-gateway/assets/2199726/77f05ebf-4329-4037-a17c-cb0046c9e387">

Reply:
Not that I know of, so I added some white spaces and a note that the variable value itself should not have white spaces. Normal line wrapping will work now.